iphone/package.json

```diff
@@ -20,7 +20,7 @@
 	"minWatchosVersion": "2.0",
 	"vendorDependencies": {
 		"xcode": ">=9.0 <=11.x",
-		"ios sdk": ">=10.0 <=13.x"
```

Comment:
@cb1kenobi  In my [PR-10932](https://github.com/appcelerator/titanium_mobile/pull/10932), I have made changes on basis of our previous history to support minimum iOS SDK and minimum XCode version  from [here](https://github.com/appcelerator/titanium_mobile/pull/10105) (to support iOS 12). So I made minimum Xcode to Xcode 9.0 and minimum iOS SDK to 10.0. Did I missed any change in [PR-10932](https://github.com/appcelerator/titanium_mobile/pull/10932)?. I think for pulling minimum iOS version [here](https://github.com/appcelerator/titanium_mobile/blob/a48d707984e88e92f05287dcd3c2588025c9c9f2/build/lib/packager.js#L88)(), we should use [minIosVersion](https://github.com/appcelerator/titanium_mobile/blob/98fa81bb2d947b4c3010a73645e5f414da7d9605/iphone/package.json#L19)(if it requires minimum iOS target as @ewanharris  as mentioned in ticket).

Reply:
@vijaysingh-axway please see my comment in the [ticket](https://jira.appcelerator.org/browse/TIMOB-27292?focusedCommentId=450137&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-450137).